lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -566,7 +564,7 @@ export class Vulnerabilities {
     vulnerability: Osv.Vulnerability,
     affected: Osv.Affected
   ): SeverityDetails {
-    let severityLevel: string | undefined;
+    let severityLevel = 'Unknown';
```

Comment:
Shouldn't be this capitalised? 
```suggestion
    let severityLevel = 'UNKNOWN';
```

Reply:
That said, let me improve this.

Reply:
Added a titleCase utility function and made severityLevel always uppercased